src/ifcsverchok/__init__.py

```diff
@@ -84,7 +96,131 @@ def make_node_list():
 reload_event = False
 
 import bpy
+import os
+from os.path import abspath, splitext
+import ifcopenshell
+from ifcsverchok.ifcstore import SvIfcStore
+from sverchok.data_structure import flatten_data
+
+class IFC_Sv_UpdateCurrent(bpy.types.Operator):
+    """Update current Sverchok node tree"""
+    bl_idname = "ifc.sverchok_update_current"
+    bl_label = "Update current node tree"
+    bl_options = {'REGISTER', 'UNDO', 'INTERNAL'}
+
+    node_group: bpy.props.StringProperty(default="")
+    force_mode: bpy.props.BoolProperty(default=False)
+
+    def execute(self, context):
+        print("#"*10, "Update current node tree")
```

Comment:
For now it's fine to center around buildings. But for the future it'd be good to also allow other infra-related spatial structure elements, such as IfcBridge. Maybe make a comment regarding that.